product_extended_segmentation/data/data.xml

```diff
@@ -22,5 +22,26 @@ self.update_material_cost_on_zero_segmentation(cr, uid, [])
             </field>
         </record>
 
+        <record model="ir.actions.server" id="update_product_segmentation_from_stock_card">
+            <field name="name">Update Segmentation on Average Product from Stock Card</field>
+            <field name="model_id" eval="ref('product.model_product_product')"/>
+            <field name="state">code</field>
+            <field name="condition"></field>
```

Comment:
@nhomar 
This Server Action is to be called on demand, it will not be triggered on `-u all` command.